app/models/payout_report.rb

```diff
@@ -0,0 +1,26 @@
+class PayoutReport < ApplicationRecord
+  has_paper_trail
```

Comment:
I don't think we want to run paper trail on this

Reply:
@yachtcaptain23 why not?  I'm thinking we definitely want to track any changes made to a payout report after it's been generated.